### PR TITLE
EWPP-1630: Add lists property to file-teaser.

### PR DESCRIPTION
**Pull request by nagyad**

### CODE REVIEW

**Review by 22Alexandra — COMMENTED** [review]
The build is failing.

**Comment by nagyad** [comment]
Waiting for this PR to be merged. https://github.com/openeuropa/oe_theme/pull/976

File: tests/Kernel/fixtures/rendering.yml [hunk 1]
```diff
@@ -657,6 +662,11 @@
       'ul.ecl-file__translation-list': 1
       'li.ecl-file__translation-item a.ecl-file__translation-download': 1
       'a.ecl-file__translation-download[href="http://example.com/document_fr.xls"]': 1
+      'div.ecl-file__taxonomy': 1
+      'dl.ecl-description-list--taxonomy': 1
+      'dt.ecl-description-list__term': 1
+      'dd.ecl-description-list__definition': 1
+      'span.ecl-description-list__tag': 2
```
Comment (imanoleguskiza) [inline tests/Kernel/fixtures/rendering.yml]: I wonder if we should also check the contents of the tags and the terms

**Review by imanoleguskiza — DISMISSED** [review]

**Review by 22Alexandra — DISMISSED** [review]

**Review by 22Alexandra — APPROVED** [review]

**Review by upchuk — APPROVED** [review]
For imanol.

**Pull request merged**